Model/Api/ShippingMethods.php

```diff
@@ -346,7 +346,7 @@ public function getShippingMethods($cart, $shipping_address)
         $startTime = $this->metricsClient->getCurrentTime();
         try {
             // get immutable quote id stored with transaction
-            list(, $quoteId) = explode(' / ', $cart['display_id']);
+            $quoteId = $cart['metadata']['immutable_quote_id'];
```

Comment:
nit: isset

Reply:
is this one necessary as it's in a try-catch already?

Reply:
Yeah, we can leave it as it is.